challenge_3.R

```diff
@@ -0,0 +1,21 @@
+#load the wages.csv file
+wages=read.csv(file="wages.csv",header=TRUE, stringsAsFactors=FALSE)
+
+#only list the years of schoool and the wage for each of the indiv in wages.csv
+wages3=wages[,3:4]
+
+#make a list of only those who have graduated from HS (12 years of school)
+HSgrads=wages3[wages3[,1]=="12",]
```

Comment:
Watch out naming things with uninformative names. If you come back later, you might not know what you were trying to do.